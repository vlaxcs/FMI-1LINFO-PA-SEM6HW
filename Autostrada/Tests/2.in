450
10 11
60 90
20 142
54 123
42 123
90 97
98 99
100 111
121 121
111 121