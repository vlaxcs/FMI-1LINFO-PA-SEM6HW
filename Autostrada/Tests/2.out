[10, 11]
[20, 142]

(0, 10)
(11, 20)
(142, 450)

27%